### PR TITLE
mpc-be: Add persistence

### PR DESCRIPTION
**Pull request by dodorz**

### CODE REVIEW

File: bucket/mpc-be.json [hunk 1]
```diff
@@ -36,7 +36,15 @@
     "pre_install": [
         "$fold = (Get-ChildItem \"$dir\" 'MPC-BE*' | Select-Object -First 1).FullName",
         "Move-Item \"$fold\\*\" \"$dir\"",
-        "Remove-Item \"$fold\" -Force -Recurse"
+        "Remove-Item \"$fold\" -Force -Recurse",
+        "if (!(Test-Path \"$persist_dir\\mpc-be64.ini\")) { New-Item \"$dir\\mpc-be64.ini\" -ItemType File | Out-Null }",
```
Comment (Ash258) [inline bucket/mpc-be.json]:
```suggestion
        "@('mpc-be64.ni', 'mpc-be.ini', 'Default.mpcpl') | ForEach-Object {",
        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType File | Out-Null }",
        "}",
```

**Comment by Ash258** [comment]
Add these into other mpc manifests #1961

File: README.md [hunk 2]
```diff
@@ -1,6 +1,3 @@
-# scoop-extras [![Build status](https://ci.appveyor.com/api/projects/status/r0bbdnvk15t1vnps?svg=true)](https://ci.appveyor.com/project/lukesampson/scoop-extras)
+# scoop-extras fork
```
Comment (Ash258) [inline README.md]: Remove REadme from this pR

File: bucket/mpc-hc-fork.json [hunk 3]
```diff
@@ -33,6 +33,15 @@
...
         }
     },
+    "pre_install": [
+        "@('mpc-hc64.ini', 'mpc-hc.ini') | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType File | Out-Null }",
+        "}"
+    ],
+    "persist": [
+        "mpc-hc64.ini",
+        "mpc-hc.ini"
```
Comment (Ash258) [inline bucket/mpc-hc-fork.json]: There is no Defaults as in other forks?

Comment (dodorz) [reply bucket/mpc-hc-fork.json]: Yes, mpc-hc-fork doesn't have the file.

**Pull request merged**